ledger-core/v2/network/consensus/gcpv2/phasebundle/metrics/metrics.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Insolar Network Ltd.
```

Comment:
move this package to network/concensus, so all concensus metrics will be in one place

Reply:
done